### ISSUE TITLE
Instructions for building

### ISSUE THREAD

**Issue by JoaquinSiabra** [pjheslin/DiogenesWeb#1]
Hello! Could you upload instructions for building the app in local? Thanks a lot

**Comment by pjheslin** [comment]
There's not really much point in running it locally.  When I was developing the app, all of the files were served locally for convenience.   But when I get the app to the point where it was almost finished, I changed it so that all static files are served from the web for speed and efficiency.  In other words most of the app's logic is contained in static JS files that are not served locally.  That makes development harder but it is a more scalable architecture.

Why do you want to run it locally?  The Diogenes desktop application has more features, if you want to run something on your computer.  DiogenesWeb is designed for phones and tablets.

**Comment by JoaquinSiabra** [comment]
Hello, thanks for replying!

I´m trying to read and process Perseus' texts, no just reading them for the study of greek. But I find a little hard to parse those XML files. The point of running Diogenes locally is debugging it and learning how you have solved the parser part :-) 

BTW, your app is great.

**Comment by pjheslin** [comment]
Ah, in that case, for your purposes, the app is already running locally -- in your browser.  I designed DiogenesWeb to offload as much processing as possible to the client.  So the code that takes the XML, parses it, and converts it to HTML is in file-display.js, which runs in your browser.  To look at it, point your desktop browser at DiogenesWeb, open up the developer tools, and choose a text.  You can use the Javascript console and debugger to see what the code does.

I'm glad you like the app!
